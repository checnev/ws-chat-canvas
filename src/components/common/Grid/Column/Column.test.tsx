import { render, unmountComponentAtNode } from 'react-dom';
import { act } from 'react-dom/test-utils';
import pretty from 'pretty';
import GridColumn from 'components/common/Grid/Column';

let container: any = null;

beforeEach(() => {
  container = document.createElement('div');
  document.body.append(container);
});

afterEach(() => {
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

const setUp = () => (
  <GridColumn
    xs={12}
    sm={10}
    md={8}
    lg={6}
    xl={4}
    xxl={2}
  >
    Grid Column Text
  </GridColumn>
);

it('should render GridColumn with size props', () => {
  act(() => {
    render(setUp(), container);
  });

  expect(
    pretty(container.innerHTML)
  ).toMatchSnapshot();
});
