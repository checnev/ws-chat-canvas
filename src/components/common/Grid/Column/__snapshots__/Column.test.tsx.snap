// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render GridColumn with size props 1`] = `"<div class=\\"gridRow__column gridRow__column_xs_12 gridRow__column_sm_10 gridRow__column_md_8 gridRow__column_lg_6 gridRow__column_xl_4 gridRow__column_xxl_2\\">Grid Column Text</div>"`;
